CONTRIBUTING.rst

```diff
@@ -0,0 +1,39 @@
+contributing
+------------
+
+in order for a pull request to be merged, please make sure it meets the following criteria:
+
+1. all unit tests pass for all supported versions of python and django.
+2. every newly added line of code is exercised by at least one unit tests.
+3. all code is compliant with PEP8 style conventions.
```

Comment:
For these requirements it may be good to point to the instructions further down in the document, and to also mention that Travis will be checking all three automatically. Maybe link to or include a screenshot of how Travis reports errors and how a contributor could use the Travis results to fixup their PR.